lib/load_aws.rb

```diff
@@ -2,14 +2,14 @@
 
 def generate_aws_yml_file
   sample_aws = {"aws"=>{"access_key_id"=>"", "secret_access_key"=>"", "region"=>""}}
-  File.open(File.join(Dir.pwd, 'aws.yml'), 'w') do |file|
+  File.open(File.join(Dir.pwd, 'config/aws.yml'), 'w') do |file|
```

Comment:
I'd prefer if you have something akin to `Rails.root` - that will get our bearings right when we are trying to navigate or load files from folders.

Reply:
Sounds good. Thinking of making a class QIt, where we can add a class method root which will return the application root path.. also going forward, anything else that can be used throughout the application can be put in here. That's the general idea. Thoughts?